### ISSUE TITLE
Example in readme doesn't work out of the box

### ISSUE THREAD

**Issue by rxhanson** [taylorgoolsby/graphql-directive-sql#2]
Thanks for making this! Should save some me some time as I get some initial tables in place on a project I'm working on. 

If I were to copy and paste your example in the readme and use node to run it, node will choke on the imports - probably something with es modules? Whatever the case, I was able to get things up and running by cloning your project and running the test.

**Comment by gaharrington** [comment]
@rxhanson I cloned this repo, but I have **not** been able to get past the issue with imports. Can you LMK what your fix was? Thanks!

**Comment by rxhanson** [comment]
I don't quite remember everything I did, but I remember having some issues. For some reason I think I just started over from scratch, did an `npm install`, compiled the test with `tsc` and ran `node <test_js_file>`.

Let me know if that doesn't work and I can try to retrace my steps.

**Comment by gaharrington** [comment]
@rxhanson Thanks for your help!

I am still getting errors with those steps:

https://share.getcloudapp.com/E0uEYEOB

**Comment by rxhanson** [comment]
@gaharrington Ah, right. I did manually install at the very least gql, and maybe a couple more libraries. I think I just did a quick npm install gql.

**Comment by taylorgoolsby** [comment]
Those import statements are part of babel. https://babeljs.io/
Babel is a preprocessor that converts non-standard JS syntax into standard syntax which `node` will understand.
Running `node generate-sql.js` assumes you have babel setup.
If you want to use new features and syntax in the Javascript language, you should take the time to learn babel. Also, lots of javascript libraries make use of babel plugins to provide useful syntax. It works with Typescript, too. You'll probably need to learn it eventually, but it can be a headache to learn right now if it's not going to give you any immediate benefits.

Just FYI, to setup babel for the example, you would need to:
1. Create a `.babelrc` file with these contents:
```
{
  "presets": [
    [
      "@babel/preset-env",
      {
        "targets": {
          "node": "current"
        }
      }
    ]
  ]
}
```
2. `npm install --dev @babel/node @babel/core @babel/preset-env`
3. `npm install graphql graphql-tag graphql-to-sql graphql-tools`
4. `npx babel-node generate-sql.js`

But don't worry about that if you're not interested in learning babel. I'll modify the example to work without babel.

To summarize, if you ever see `import x from 'x'`, that's babel syntax. You can convert it manually to regular JS by replacing it with `const x = require('x')`.

8dec231d600fc28127971a7f08484ebe89da4e9f
